### ISSUE TITLE
Getting aution info and bidding stopped working

### ISSUE THREAD

**Issue by alexpilotti** [ruippeixotog/ebay-snipe-server#5]
Trying to retrieve bid info on ebay.com on any auction, returns non existing data.
Everything used to work until recently. I wonder if changes in the Ebay site caused the scraping to stop working:

`http://<host>/auction/<id>`

```
{
    "ended": true,
    "endingAt": "1970-01-01T00:00:00.000Z",
    "location": null,
    "bidCount": 0,
    "shippingCost": "UNK 0.0",
    "currentBid": null,
    "id": "<id>",
    "buyNowPrice": null,
    "defaultCurrency": "UNK",
    "title": "",
    "seller": {
        "id": null,
        "feedback": 0,
        "positivePercentage": 100
    },
    "thumbnailUrl": null
}
```

**Comment by ruippeixotog** [comment]
Hi @alexpilotti, thanks for submitting this! I can confirm that it's not working now. This is probably due to changes on the ebay.com site, as you said. I'll take a look at this later this week.

**Comment by ruippeixotog** [comment]
This should be fixed in 4dbb560adfec119e056593a45c8430409f0bf506. Let me know if the problem persists :)
